cpp-package/example/utils.h

```diff
@@ -27,6 +27,15 @@
 
 using namespace mxnet::cpp;
 
+#define TRY \
+  try {
+#define CATCH \
+  } catch(...) { \
+    LG << "Status: FAIL";\
```

Comment:
catch only dmlc::Error. For non backend exceptions MXGetLastError will be empty.

Reply:
Fixed.